packages/react/src/SelectionArea.tsx

```diff
@@ -13,8 +13,16 @@ export interface SelectionAreaProps extends Omit<Partial<SelectionOptions>, 'bou
     onStop?: SelectionEvents['stop'];
 }
 
+const SelectionContext = createContext<Object | undefined>(undefined);
```

Comment:
You can use the type `SelectionArea` instead of `Object` to make it type-safe :)

Reply:
i did it, i hope it's ok